portfolio/src/main/webapp/index.html

```diff
@@ -7,22 +7,22 @@
     <script src="script.js"></script>
   </head>
   <body>
+    
     <div id="content">
-      <h1>Kaleab Gebremichael</h1>
-
-      <div class="navbar">
-        <a href="#about">About me</a>
-        <a href="#work">Work Experience</a>
-        <a href="#hobbies">Hobbies</a>
-        <a href="#contact">Contact Me</a>
-      </div>
+        <div class="navbar">
+            <a href="#about">About me</a>
+            <a href="#work">Work Experience</a>
+            <a href="#hobbies">Hobbies</a>
+            <a href="#contact">Contact Me</a>
+        </div>
+      <h1 class="headers"> Kaleab Gebremichael</h1>
 
       <p id="about"> My name is Kaleab Gebremichael and I'm a rising junior at Vanderbilt. I am majoring in Computer Science and Mathematics.</p>
       <img class="photo" src="Images/naruto-1.png">
       
       <hr>
 
-      <h3 id="work">Work Experience</h3>
+      <h3 class="headers" id="work">Work Experience</h3>
```

Comment:
Looks like you put this on all, and only on the `<h3>` tags. You can just use that selector in the css, then you won't need these classnames at all.
```
h3 {
  margin-left: 75px;
}
```